src/main/java/uk/gov/hmcts/sptribs/document/content/DocmosisTemplateConstants.java

```diff
@@ -22,117 +22,125 @@ public final class DocmosisTemplateConstants {
-        + " is successful. Mr/Mrs …… is eligible for compensation. The CICA must now consider if any compensation should be awarded.\n"
-        + "2.     Mr/ Mrs …….application is now returned to the CICA for further decision making and action.\n"
-        + "3.     Summary of reasons for the decision";
+    public static final String ELIGIBILITY_MAIN_CONTENT = """
+        1.     The appeal against the CICA’s Review decision dated…..under paragraph….. is successful. Mr/Mrs …… is eligible for \
+        compensation. The CICA must now consider if any compensation should be awarded.
+        2.     Mr/ Mrs …….application is now returned to the CICA for further decision making and action.
+        3.     Summary of reasons for the decision""";
 
-    public static final String QUANTUM_MAIN_CONTENT = "1.    The appeal is allowed.\n"
-        + "2.     The appeal against the CICA's Review Decision dated........under paragraphs 32&33 is successful.\n"
-        + "3.     Mr/Ms B is entitled to a total award of…………  Details of this award are set out in the calculation overleaf.\n"
-        + "4.     Summary of reasons for decision";
+    public static final String QUANTUM_MAIN_CONTENT = """
+        1.    The appeal is allowed.
+        2.     The appeal against the CICA's Review Decision dated........under paragraphs 32&33 is successful.
+        3.     Mr/Ms B is entitled to a total award of…………  Details of this award are set out in the calculation overleaf.
+        4.     Summary of reasons for decision""";
 
-    public static final String RULE27_MAIN_CONTENT = "1.     Rule 27 (4) provides that the Tribunal may make a decision which disposes of"
-        + " proceedings without a hearing.\n"
-        + "2.     The Tribunal has decided to make a decision without a hearing because\n"
-        + "3.     The appeal is\n"
-        + "4.     Decision;\n"
-        + "5.     The Tribunal made the following findings of facts\n"
-        + "6. Any party may make a written application to the Tribunal for the decision to be reconsider at a hearing. Any application "
-        + "must be received within one month after the date on which the Tribunal sent notice of the decision to the party making the "
-        + "application.";
+    public static final String RULE27_MAIN_CONTENT = """
+        1.     Rule 27 (4) provides that the Tribunal may make a decision which disposes of proceedings without a hearing.
+        2.     The Tribunal has decided to make a decision without a hearing because
+        3.     The appeal is
+        4.     Decision;
+        5.     The Tribunal made the following findings of facts
+        6.     Any party may make a written application to the Tribunal for the decision to be reconsider at a hearing. \
+        Any application must be received within one month after the date on which the Tribunal sent notice \
+        of the decision to the party making the application.""";
 
     public static final String ME_DMI_MAIN_CONTENT =
```

Comment:
Can the unwanted constants from the EventConstants file be removed as part of this?